### PR TITLE
Fix Remove Legacy Page #3013

### PR DESCRIPTION
**Pull request by marvinsanchez1**
Fix issue on Remove Legacy Page #3167 

I'm terribly sorry that I took so long to fix the issue. I was having issues with my computer but I hope this fixes the issue.

### CODE REVIEW

File: frontend/src/pages/legacy/en-US.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Legacy
+
+Hello there! Once upon a time, you installed an add-on called Test Pilot for Firefox.
+
+Well the Test Pilot program for trying experimental features in Firefox is back, and we thought you might like to check it out.
+The old add-on has been uninstalled for you. If you&apos;d like the new one, just click the link.
```
Comment (fzzzy) [inline frontend/src/pages/legacy/en-US.md]: Please link this to https://testpilot.firefox.com.

**Review by fzzzy — CHANGES_REQUESTED** [review]
This looks good except for one small nit. Thank you!

File: frontend/src/pages/legacy/en-US.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Legacy
```
Comment (fzzzy) [inline frontend/src/pages/legacy/en-US.md]: Oops, one more bug here: This says "Legacy", but the integration test checks for it saying "Legacy Test Pilot", so the integration test fails. Change this to say "Legacy Test Pilot".

Comment (marvinsanchez1) [reply frontend/src/pages/legacy/en-US.md]: Got it ! - Thank you for pointing that out

File: frontend/src/pages/legacy/en-US.md (hunk 1 above)
Comment (marvinsanchez1) [reply frontend/src/pages/legacy/en-US.md]: You want me to delete the line 3-6 and replace it with the link?
[you can find more about it here](https://testpilot.firefox.com). ? or just the link

Comment (fzzzy) [reply frontend/src/pages/legacy/en-US.md]: Just replace the word "link" with `[link](https://testpilot.firefox.com)`

**Comment by chuckharmston** [comment]
Rebooted the integration tests, looked like an intermittent failure.

**Comment by meandavejustice** [comment]
@fzzzy Looks like this is good and passing integration tests now.

**Review by fzzzy — APPROVED** [review]

**Pull request merged**